python/lsst/ts/scriptqueue/script_info.py

```diff
@@ -55,37 +55,48 @@ class ScriptInfo:
         Configuration data as a YAML encoded string.
     descr : `str`
         A short explanation of why this script is being run.
+    log_level : `int` (optional)
+        Log level for the script, as a Python logging level.
+        Specify 0 (the default) to leave the level unchanged.
+    pause_checkpoint : `str` (optional)
+        Checkpoint(s) at which to pause, as a regular expression.
+        No checkpoints if blank; all checkpoints if ".*".
+    stop_checkpoint : `str` (optional)
+        Checkpoint(s) at which to stop, as a regular expression.
+        No checkpoints if blank; all checkpoints if ".*".
     verbose : `bool` (optional)
         If True then print log messages from the script to stdout.
     """
-    def __init__(self, log, remote, index, seq_num, is_standard, path, config, descr, verbose=False):
+    def __init__(self, log, remote, index, seq_num, is_standard, path, config, descr,
+                 log_level=0, pause_checkpoint="", stop_checkpoint="", verbose=False):
         self.log = log
         self.remote = remote
         self.index = int(index)
         self.seq_num = int(seq_num)
         self.is_standard = bool(is_standard)
         self.path = str(path)
         self.config = config
+        self.log_level = log_level
+        self.pause_checkpoint = pause_checkpoint
+        self.stop_checkpoint = stop_checkpoint
         self.descr = descr
         self.verbose = verbose
+        # the most recent state reported by the Script
+        # or 0 if the script is not yet loaded
         self.script_state = 0
-        """Most recent state reported by the Script, or 0 if the script
-        is not yet loaded."""
+        # Delay between when the script sent state and it was received (sec)
         self.state_delay = 0
-        """Delay between time script sent state and it was received."""
+        # Time at which the script process was started. 0 before that.
         self.timestamp_process_start = 0
-        """Time at which the script process was started. 0 before that."""
+        # Time at which the _configure method was started. 0 before that.
+        # Note: under unusual circumstances the _configure method may fail
+        # before the configure command is sent to the task.
         self.timestamp_configure_start = 0
-        """Time at which the _configure method was started. 0 before that.
-
-        Note: under unusual circumstances the _configure method may fail
-        before the configure command is sent to the task.
-        """
+        # Time at which the configure command finished (succeeded or failed).
+        # 0 before that.
         self.timestamp_configure_end = 0
-        """Time at which the configure command finished (succeeded or failed).
-        0 before that."""
+        # Time at which the script started running. 0 before that."""
```

Comment:
I will also update the other strings. I originally used strings because I thought sphinx would see them, but as we know it does not for instance variables.